share/wake/lib/core/double.wake

```diff
@@ -13,65 +13,67 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 package wake
 
 # The Double type is an IEEE 754 64-bit precision number.
 # Any wake literal which include a decimal point or an exponent is parsed into a Double.
 
+
 # Unary absolute value operator for a Double.
 # dabs (+. 2.5) = 2.5
 # dabs (-. 2.5) = 2.5
-export def dabs (x: Double): Double =
-    (\_ prim "dabs") x
+export def dabs (x: Double): Double = (\_ prim "dabs") x
```

Comment:
For the first pass (and probably the forseeable future) I'd like to avoid rules that only apply to top-level as much as possible. When emitting the code there currently isn't a great way to know what level you are on (you can guess via nest level but that is a bit hacky). This top-level stuff only worked for comments since they have a separate dedicated subsystem and even that has a small hack in it